### PR TITLE
Add basic read and write methods

### PR DESCRIPTION
**Pull request by sfinkens**
I added basic read and write methods using satpy's `avhrr_l1b_gaclac` reader and the CF writer. Example:

```python
from pygac_fdr import read_gac, PygacFdrNetcdfWriter

scene = read_gac('/data/avhrr_gac/data/NSS.GHRR.NM.D04111.S2305.E0050.B0947778.GC',
                 reader_kwargs={'tle_dir': '/data/avhrr_gac/tle',
                                'tle_name': 'TLE_%(satname)s.txt',
                                # TODO: 'calib_coefs': my_coefs
                                })
writer = PygacFdrNetcdfWriter()
writer.write(scene)

```

### CODE REVIEW

**Comment by sfinkens** [comment]
Example ncdump:
<details>

```
netcdf avhrr_gac_fdr_noaa17_20040420_230513_20040421_005047 {
dimensions:
	y = 12668 ;
	x = 409 ;
	num_flags = 7 ;
variables:
	double acq_time(y) ;
		acq_time:_FillValue = NaN ;
		acq_time:long_name = "Mean scanline acquisition time" ;
		acq_time:units = "seconds since 2004-04-20 23:05:13.480000" ;
		acq_time:calendar = "proleptic_gregorian" ;
	double longitude(y, x) ;
		longitude:_FillValue = NaN ;
		longitude:name = "longitude" ;
		longitude:standard_name = "longitude" ;
		longitude:units = "degrees_east" ;
	double latitude(y, x) ;
		latitude:_FillValue = NaN ;
		latitude:name = "latitude" ;
		latitude:standard_name = "latitude" ;
		latitude:units = "degrees_north" ;
	double channel_1(y, x) ;
		channel_1:_FillValue = NaN ;
		channel_1:calibration = "reflectance" ;
		channel_1:modifiers = "" ;
		channel_1:resolution = 1050LL ;
		channel_1:standard_name = "toa_bidirectional_reflectance" ;
		channel_1:units = "%" ;
		channel_1:wavelength = 0.58, 0.63, 0.68 ;
		channel_1:coordinates = "latitude acq_time longitude" ;
	double channel_2(y, x) ;
		channel_2:_FillValue = NaN ;
		channel_2:calibration = "reflectance" ;
		channel_2:modifiers = "" ;
		channel_2:resolution = 1050LL ;
		channel_2:standard_name = "toa_bidirectional_reflectance" ;
		channel_2:units = "%" ;
		channel_2:wavelength = 0.725, 0.8625, 1. ;
		channel_2:coordinates = "latitude acq_time longitude" ;
	double channel_3a(y, x) ;
		channel_3a:_FillValue = NaN ;
		channel_3a:calibration = "reflectance" ;
		channel_3a:modifiers = "" ;
		channel_3a:resolution = 1050LL ;
		channel_3a:standard_name = "toa_bidirectional_reflectance" ;
		channel_3a:units = "%" ;
		channel_3a:wavelength = 1.58, 1.61, 1.64 ;
		channel_3a:coordinates = "latitude acq_time longitude" ;
	double channel_3b(y, x) ;
		channel_3b:_FillValue = NaN ;
		channel_3b:calibration = "brightness_temperature" ;
		channel_3b:modifiers = "" ;
		channel_3b:resolution = 1050LL ;
		channel_3b:standard_name = "toa_brightness_temperature" ;
		channel_3b:units = "K" ;
		channel_3b:wavelength = 3.55, 3.74, 3.93 ;
		channel_3b:coordinates = "latitude acq_time longitude" ;
	short channel_4(y, x) ;
		channel_4:_FillValue = -32767s ;
		channel_4:calibration = "brightness_temperature" ;
		channel_4:modifiers = "" ;
		channel_4:resolution = 1050LL ;
		channel_4:standard_name = "toa_brightness_temperature" ;
		channel_4:units = "K" ;
		channel_4:wavelength = 10.3, 10.8, 11.3 ;
		channel_4:coordinates = "latitude acq_time longitude" ;
		channel_4:add_offset = 273.15 ;
		channel_4:scale_factor = 0.01 ;
	double channel_5(y, x) ;
		channel_5:_FillValue = NaN ;
		channel_5:calibration = "brightness_temperature" ;
		channel_5:modifiers = "" ;
		channel_5:resolution = 1050LL ;
		channel_5:standard_name = "toa_brightness_temperature" ;
		channel_5:units = "K" ;
		channel_5:wavelength = 11.5, 12., 12.5 ;
		channel_5:coordinates = "latitude acq_time longitude" ;
	string num_flags(num_flags) ;
	double qual_flags(y, num_flags) ;
		qual_flags:_FillValue = NaN ;
		qual_flags:long_name = "Scanline Quality Flags" ;
		qual_flags:resolution = 1050LL ;
		qual_flags:coordinates = "acq_time" ;
	double sensor_azimuth_angle(y, x) ;
		sensor_azimuth_angle:_FillValue = NaN ;
		sensor_azimuth_angle:resolution = 1050LL ;
		sensor_azimuth_angle:standard_name = "sensor_azimuth_angle" ;
		sensor_azimuth_angle:units = "degrees" ;
		sensor_azimuth_angle:coordinates = "latitude acq_time longitude" ;
	double sensor_zenith_angle(y, x) ;
		sensor_zenith_angle:_FillValue = NaN ;
		sensor_zenith_angle:resolution = 1050LL ;
		sensor_zenith_angle:standard_name = "sensor_zenith_angle" ;
		sensor_zenith_angle:units = "degrees" ;
		sensor_zenith_angle:coordinates = "latitude acq_time longitude" ;
	double solar_azimuth_angle(y, x) ;
		solar_azimuth_angle:_FillValue = NaN ;
		solar_azimuth_angle:resolution = 1050LL ;
		solar_azimuth_angle:standard_name = "solar_azimuth_angle" ;
		solar_azimuth_angle:units = "degrees" ;
		solar_azimuth_angle:coordinates = "latitude acq_time longitude" ;
	double solar_zenith_angle(y, x) ;
		solar_zenith_angle:_FillValue = NaN ;
		solar_zenith_angle:resolution = 1050LL ;
		solar_zenith_angle:standard_name = "solar_zenith_angle" ;
		solar_zenith_angle:units = "degrees" ;
		solar_zenith_angle:coordinates = "latitude acq_time longitude" ;
	double sun_sensor_azimuth_difference_angle(y, x) ;
		sun_sensor_azimuth_difference_angle:_FillValue = NaN ;
		sun_sensor_azimuth_difference_angle:resolution = 1050LL ;
		sun_sensor_azimuth_difference_angle:standard_name = "angle_of_rotation_from_solar_azimuth_to_platform_azimuth" ;
		sun_sensor_azimuth_difference_angle:units = "degrees" ;
		sun_sensor_azimuth_difference_angle:coordinates = "latitude acq_time longitude" ;

// global attributes:
		:creation_time = "2020-05-11T18:16:13.508107" ;
		:end_time = "2004-04-21 00:50:47.980000" ;
		:gac_file = "NSS.GHRR.NM.D04111.S2305.E0050.B0947778.GC" ;
		:orbit_number = 9477LL ;
		string :orbital_parameters_tle = "1 27453U 02032A   04111.87958483  .00000188  00000-0  10197-3 0  8815\n", "2 27453  98.7030 184.7706 0011999 195.4825 164.5990 14.23517495 94756\n" ;
		:platform_name = "noaa17" ;
		:sensor = "avhrr-3" ;
		:start_time = "2004-04-20 23:05:13.480000" ;
		:history = "Created by pytroll/satpy on 2020-05-11 16:16:13.516450" ;
		:Conventions = "CF-1.7" ;
```

</details>

**Comment by mraspaud** [comment]
Do we want to add a main script to this PR ?

**Comment by sfinkens** [comment]
Good idea!

File: pygac_fdr/__init__.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                    'orbit_number']
+    dataset_names = {
+        '1': 'channel_1',
+        '2': 'channel_2',
+        '3': 'channel_3',
+        '3a': 'channel_3a',
+        '3b': 'channel_3b',
+        '4': 'channel_4',
+        '5': 'channel_5',
+    }
```
Comment (sfinkens) [inline pygac_fdr/__init__.py:50]: Move outside the class

**Pull request merged**